modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties

```diff
@@ -2319,7 +2319,12 @@ boost-must-be-greater-than-or-equal-to-0=Boost must be greater than or equal to
 bootstrap-mlockall=Bootstrap Mlockall
 bootstrap-mlockall-help=This configuration only applies to EMBEDDED (Development) mode. If this is checked, the process address space is locked into RAM, preventing any Elasticsearch memory from being swapped out.
 border=Border
+border-accent-colors=Border Accent Colors
```

Comment:
Add all your new language keys to the end of the file, and let the sorting be done by the buildLang task in a separate commit. This will make it much easier to handle rebasing when necessary.

Reply:
Thanks for the hint, Evan!